samples/drivers/i2s/i2s_codec/sample.yaml

```diff
@@ -0,0 +1,13 @@
+sample:
+  name: codec sample
+tests:
+  sample.drivers.i2s.codec:
+    tags: i2s
+    filter: dt_alias_exists("i2s_codec")
```

Comment:
this filter will make the ci skip this case, and as you have add the platform_allow, either add this alias to dts or remove it.

Reply:
Will remove then.